src/unstable/components/Form/widgets/PasswordWidget.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { FormWidgetProps } from '../../../unstable-typings';
+
+const STRENGTH_TITLES = ['Very weak', 'Weak', 'Good', 'Strong', 'Very strong'];
+const STRENGTH_STYLES = [
+	{ width: 0 },
+	{ width: '25%', backgroundColor: 'orange' },
+	{ width: '50%', backgroundColor: 'yellow' },
+	{ width: '75%', backgroundColor: 'green' },
+	{ width: '100%', backgroundColor: 'darkgreen' },
+];
+
+const PasswordStrengthContainer = styled.div`
+	padding-bottom: 4px;
+	position: relative;
+	margin-bottom: 30px;
+	font-size: 90%;
+`;
+
+const PasswordStrengthMeter = styled.div`
+	transition: all ease-out 150ms;
+	position: absolute;
+	height: 4px;
+	background-color: red;
+	min-width: 5px;
+	bottom: 0;
+`;
+
+export interface PasswordStrengthProps {
+	password?: string;
+}
+
+const PasswordStrength = ({ password }: PasswordStrengthProps) => {
+	const [strengthScore, setStrengthScore] = React.useState();
+
+	React.useEffect(() => {
+		// @ts-ignore If you wish to show a stength meter, you need to load and set `zxcvbn` to a window variable by yourself.
+		const zxcvbn = window.RENDITION_ZXCVBN;
```

Comment:
I think this could just be `window. zxcvbn`

Reply:
It is just a precaution in case the library itself uses some globals. I might be overthinking it though, I guess we can simplify it 😄